monai/transforms/io/array.py

```diff
@@ -335,6 +337,8 @@ def __init__(
         )
 
         self.output_ext = output_ext.lower() or output_format.lower()
+        if isinstance(writer, str):
+            writer, _ = optional_import("monai.data", name=f"{writer}")
```

Comment:
Maybe we can use `locate` or `importlib` to user extended writer of `ImageWriter`?
For example `writer="custom.writer.MyWriter"`.

Thanks.

Reply:
thanks, i've updated to use two searches (is there a simpler approach?) https://github.com/Project-MONAI/MONAI/blob/9a474dabe47b0ebf1a2c06874d9aa832cf603fb4/monai/transforms/io/array.py#L341-L347